### DISCUSSION TITLE
Disable Shorts on Youtube

### DISCUSSION THREAD

**Discussion by crayonware** [PoomSmart/YTABConfig#96] [General]
After spending countless hours searching on Reddit and Github, I've come up with my own solution to disable YT shorts with A/B flags. I'm not sure which ones are helpful and which ones are useless so leave a comment if you find something that can be helpful until we have a final list of flags to use. 

I've only tested this on iOS so if someone could please try this out for Android and let me know if it's working and I will update this post accordingly.

(After enabling the flags, restart YT, refresh the homepage and remove any shorts that are still lingering on the homepage, they shouldn't come back after that.)

enableResumeToShorts: 0
enableShortsTargeted: 0
iosEnableWatchToShortsTransition: 0
iosShortsPlayerCheckDeactivePlaybackAfterSubviewLayout: 0
mainAppCoreClientIosRemoveShortsInPivotFlagForIphoneKillswitch: 1
shortsClientGlobalConfigEnableSoundButtonV2: 0
shortsConsumptionClientGlobalConfigEnableLogImpressionsOnPivotActionBarButtons: 0
shortsConsumptionClientGlobalConfigIosEnableResumeToShortsOnAds: 0
shortsConsumptionClientGlobalConfigIosEnableShortsProminentNavigationBar: 0
shortsConsumptionClientGlobalConfigIosEnableShortsProminentNavigationIcons: 0
shortsConsumptionClientGlobalConfigIosRemoveStoriesEventHandler: 1
enableHidingMiniplayerOnBrowseBasedShortsPivotPage: 0
enableIosShortsAgeGating: 1
enableIosSkoverlayShorts: 0
iosDisableOnDeviceAndRefinementInShorts: 1
searchUiSharedEnableTitleRendererInGetShortsSourceVideoResponse: 0
shortsClientGlobalConfigEnableIosEndscreenPivotFix: 0
shortsClientGlobalConfigEnableIosShortsPivotToPivot: 0
shortsConsumptionClientGlobalConfigEnableNoHistoryUserClient: 1

**Comment by boyntonj** [comment]
Can confirm it is one of these, likely the last one: 

enableResumeToShorts: 0
enableShortsTargeted: 0
iosEnableWatchToShortsTransition: 0
mainAppCoreClientIosRemoveShortsInPivotFlagForIphoneKillswitch: 1
